rosdep/base.yaml

```diff
@@ -7961,6 +7961,10 @@ texmaker:
   gentoo: [app-office/texmaker]
   nixos: [texmaker]
   ubuntu: [texmaker]
+tiled:
```

Comment:
Would you mind adding the arch/alpine/opensuse rules as suggested by the tooling? Otherwise this looks fine